traptor/traptor.py

```diff
@@ -213,6 +219,11 @@ def _setup(self):
         # Create the locations_rule dict if this is a locations traptor
         self.locations_rule = {}
 
+    @retry(wait=wait_exponential(multiplier=1, max=10),
+           stop=stop_after_attempt(3),
+           reraise=True,
```

Comment:
Is the `reraise` needed here?  My understanding is that the reraise is for if you want to handle the case when the retry fails... and do something after that.